src/CurveLPOracle.sol

```diff
@@ -110,6 +113,7 @@ contract CurveLPOracle {
         require(_pool != address(0), "CurveLPOracle/invalid-pool");
         uint256 _ncoins = _orbs.length;
         pool   = _pool;
+        src    = CurvePoolLike(_pool).lp_token();
```

Comment:
Here is where we need to think about some standardization. In the Uni V2 case, the same contract holds both the token balance+swap logic as well as the LP token implementation. In Curve's case they are separate contracts. So which one should be `src` here? cc @brianmcmichael